app/src/main/java/org/oppia/app/walkthrough/WalkthroughViewModel.kt

```diff
@@ -0,0 +1,9 @@
+package org.oppia.app.walkthrough
+
+import androidx.databinding.ObservableField
+import org.oppia.app.viewmodel.ObservableViewModel
+import javax.inject.Inject
+
+class WalkthroughViewModel @Inject constructor() : ObservableViewModel() {
+  var currentProgress = ObservableField(0)
```

Comment:
val currentProgress = ObservableField(0)

`val` should be used instead of `var` whereable possible.